src/main/java/sirius/pasta/noodle/SiriusClassAliasProvider.java

```diff
@@ -62,11 +64,12 @@ public void collectAliases(BiConsumer<String, Class<?>> consumer) {
         consumer.accept("CallContext", CallContext.class);
         consumer.accept("UserContext", UserContext.class);
         consumer.accept("WebContext", WebContext.class);
-        consumer.accept("Product", Product.class);
         consumer.accept("Injector", Injector.class);
         consumer.accept("Strings", Strings.class);
         consumer.accept("Files", Files.class);
         consumer.accept("Hasher", Hasher.class);
         consumer.accept("Amount", Amount.class);
+        consumer.accept("ContentHelper", ContentHelper.class);
+        consumer.accept("Product", Product.class);
```

Comment:
this has been removed above?

Reply:
Had clashes with the one in OX so reintroduces it here. Not sure if its really needed in combination with the fully qualified imports in the templates.